frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/review-tab/review-tab.scss

```diff
@@ -1,7 +1,54 @@
-.kubevirt-create-vm-modal__review-tab-title {
-  margin-bottom: 1em;
+@import '~@patternfly/patternfly/sass-utilities/all';
+
+.kubevirt-create-vm-modal__review-tab-section {
+  padding-bottom: var(--pf-global--spacer--xl);
+  &__title {
+    margin-top: var(--pf-global--spacer--lg);
+    margin-bottom: var(--pf-global--spacer--md);
+  }
+  &__content {
+    &.pf-c-content ol {
+      --pf-c-content--ol--MarginLeft: 0;
+      --pf-c-content--ol--PaddingLeft: var(--pf-global--spacer--md);
+    }
+  }
+}
+
+.kubevirt-create-vm-modal__review-tab__data-list {
+  display: grid;
+  grid-template-columns: 1fr;
+
+  @media screen and (min-width: 576px) {
+    grid-template: auto / auto 1fr;
+    grid-column-gap: var(--pf-global--spacer--2xl);
+    grid-row-gap: var(--pf-global--spacer--sm);
+  }
+}
+
+dt {
```

Comment:
Hi @pcbailey, it looks this changes the styling of all `dt` and `dd` elements in the console, breaking the layout of our detail pages. This needs to be scoped to only the context where it is used. We should be following http://getbem.com/ naming and ideally use a class name for these styles.